examples/25-Chronos-2-examples.ipynb

```diff
@@ -0,0 +1,572 @@
+{
```

Comment:
<p>The interval coverage  <code>mic</code>  (e.g. the share of actuals that are inside the prediction interval, we expect 80% to be inside) and interval width <code>miw</code> (the height / width of the interval) could be nice to show here :) I find the quantile loss not always very intuitive.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">from darts.metrics import mic, miw

mic_prob = mic(val_energy, pred_cov, q_interval=(0.1, 0.9))
miw_prob = miw(val_energy, pred_cov, q_interval=(0.1, 0.9))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/unit8co/darts/pull/2944/discussion/#comment-2531865948'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>